protocols/relay/src/behaviour/handler.rs

```diff
@@ -716,6 +665,14 @@ impl ConnectionHandler for Handler {
             }
         }
 
+        // Process protocol requests
+        if let Poll::Ready((_, worker_res)) = self.protocol_futs.poll_unpin(cx) {
```

Comment:
```suggestion
        if let Poll::Ready(((), worker_res)) = self.protocol_futs.poll_unpin(cx) {
```

This is the ID, right? In case we ever change it away from `()` we get a helpful compiler error here.